src/Backend/Compile.hs

```diff
@@ -40,6 +42,22 @@ supportedLlvmVersions = makeVersion . (: [minorVersion]) <$> supportedMajorVersi
   where minorVersion = 0
         supportedMajorVersions = [maxLlvmVersion, maxLlvmVersion - 1 .. minLlvmVersion]
 
+-- | llvm-config is not available in current LLVM distribution for windows, so we
+-- need use @clang -print-prog-name=clang@ to get the full path of @clang@.
+--
+-- We simply assume that @clang.exe@ already exists in @%PATH%@.
+--
+clangBinPath :: String -> IO FilePath
+clangBinPath command = trim <$> checkClangExists trySuffixes
```

Comment:
Perhaps inline the argument since it's only called with `"clang"`?

Reply:
Fixed.